docs/success-stories/success-story-1/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: success-story-page
+show_sidebar: false
+permalink: /success-stories/success-story-The-Digital-Slide-Archive-and-TCGA/
+
+# Banner Content
+# =================================
+title: Success Stories
+subtitle: The Digital Slide Archive and TCGA
+hero_image: /assets/img/home-callouts/success_stories.jpg
+
+# Story
+# =================================
+story:
+ - title: The Digital Slide Archive and TCGA
+   link-id: The-Digital-Slide-Archive-and-TCGA
+   description-para-1: "The Cancer Digital Slide Archive is a website that hosts whole-slide images of pathologic specimens associated with the TCGA studies. At over 10,000 images and 20 Terabytes, the CDSA has enabled a number of the TCGA Analysis Working Group Studies [sources], as well as other investigator-driven studies that utilize the digital pathology images [sources]<br><br>
```

Comment:
The `[sources]` should be links to the references.  @dgutman, can you indicate what sources should be referenced?

Reply:
Here are the references for this section:

The Cancer Genome Atlas Research Network Comprehensive and integrated genomic characterization of adult soft tissue sarcomas, Cell, 171(4), pp. 950-965, November 2017, PMID: 29100075

DA Gutman, M Khalilia , S Lee , M Nalisnik , Z Mullen , J Beezley , D Chittajallu , D Manthey, LAD Cooper, The Digital Slide Archive: A Software Platform for Management, Integration and Analysis of Histology for Cancer Research, Cancer Research, 77(21), pp. 75-78, November 2017, PMID: 29092945

DA Gutman, J Cobb, D Somanna, Y Park, F Wang, T Kurc, JH Saltz, DJ Brat, LAD Cooper, Cancer Digital Slide Archive: an informatics resource to support integrated in silico analysis of TCGA pathology data, Journal of the American Medical Informatics Association JAMIA, 20(6), pp. 1091-98, November 2013, PMID: 23893318

Reply:
> I notice that the "Contact" menu (which, when hovered, opens to show Visit Contact Page and Show Email) is itself clickable but just goes to the page you are on. I would expect that it wouldn't be a link by itself.

Fixed in f144c1a676e867e6fdcf90bfa73c44dbeb74e460

Reply:
added these references in story page under 'references' heading.
and removed '[sources]' from story text.
Fixed in 091dc6491eaa4546a02532b97377fac8d2032795